src/bsoncxx/types.hpp

```diff
@@ -346,10 +346,6 @@ BSONCXX_INLINE bool operator==(const b_date& lhs, const b_date& rhs) {
 ///
 /// A BSON null value.
 ///
-/// @warning
-///   This BSON type is used internally by the MongoDB server - use by clients
-///   is discouraged.
-///
```

Comment:
bsonspec.org isn't clear what's "discouraged".  Should I remove the boilerplate everywhere?  Or are there actually discouraged (but not deprecated) types?